sink/datadog.go

```diff
@@ -126,7 +138,8 @@ func NewDatadog(monitorId string, opts, tags map[string]string, httpClient *http
 
 		case "api-compress":
 			d.compress = blip.Bool(v)
-
+		case "send-counter-delta":
+			d.sendCounterDelta = blip.Bool(v)
```

Comment:
Why this option if only delta counter is valid for DD?

Reply:
I asked for this to be switchable in order to help with transitioning. Being able to take newer blip versions but not have this turned on yet would be helpful for that.

Reply:
Why not stay on 1.0 and flip versions when ready? Because even if you upgrade to 1.1, since these vars aren't dynamic, you'll have to restart Blip to switch to the new, correct behavior (delta counters).

Reply:
Ok I will remove it in that case.